.github/workflows/CI.yml

```diff
@@ -0,0 +1,36 @@
+#Names GitHub action
+name: CI
+
+#Specifies when this action runs
+on:
+  push:                           #On push to either
+    branches:                     # Master branch or
+      - master                    # CI-Inclusion branch
+      - ci-inclusion2
+    pull_request:                 #On pull-request to
+      - master                    # Master branch
+
+#Describes job that this action performs
+jobs:
+  build:
+
+    #Runs on standard ubuntu VM
+    runs-on: ubuntu-latest
+
+    #Specifies which docker container to grab
+    container:
+      image: vlamdock/riskv #This is from our team members dockerhub acct
```

Comment:
Any functional alternatives?

Reply:
> Any functional alternatives?

What are you looking for? Something other than a docker container at dockerhub? I admit that I'd rather have everything hosted at github, but that doesn't seem to be supported. We've been using something similar for snek CI:

[snek.yml](https://github.com/keith-packard/snek/blob/master/.github/workflows/snek.yml)

I think what we need to do is create a picolibc dockerhub account to host a container image for this so that it doesn't depend on a specific user account.